react-app/src/hooks/api/usePropertiesApi.ts

```diff
@@ -179,6 +184,25 @@ const usePropertiesApi = (absoluteFetch: IFetch) => {
     return parsedBody as ImportResult[];
   };
 
+  const getLinkedProjectsToProperty = async (propertyId: PropertyId) => {
+    console.log('propertyId', propertyId);
+    const queryParams = new URLSearchParams();
```

Comment:
You don't need to construct this here, the absoluteFetch.get takes params as its second argument.